.github/workflows/build-test-pytest.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
 
   build:
     strategy:
+      fail-fast: false
```

Comment:
I don't think it is useful enough to justify the carbon footprint it would generate. IMO we shouldn't keep it, but I'm fine with using it punctually if it helps you debugging your PR :)